### PR TITLE
Fix lintdiff bug

### PR DESCRIPTION
**Pull request by jianyexi**

### CODE REVIEW

File: src/momentOfTruth.ts [hunk 1]
```diff
@@ -180,7 +180,9 @@ export async function lintDiff(utils: TypeUtils, devOps: TypeDevOps) {
   const pr = await devOps.createPullRequestProperties(cli.defaultConfig());
   const configsToProcess = await utils.getConfigFilesChangedInPR(pr);
 
-  tagsMap = await utils.getTagsFromChangedFile(await utils.getChangedFilesFromPR(pr));
+  tagsMap = await utils.getTagsFromChangedFile(
```
Comment (raych1) [inline src/momentOfTruth.ts:183]: What's the difference between these functions `getChangedFilesFromPR` and `getFilesChangedInPR`?

File: src/tests/momentOfTruthTest.ts [hunk 2]
```diff
@@ -51,9 +53,13 @@ class MomentOfTruthTest {
     let stub4 = sinon.stub(devOps, "createPullRequestProperties").returns({
       workingDir: cwd + "/src/tests/Resource/momentOfTruth/old",
       checkout: () => "true",
-      diff: () => {
+      diff: async () => {
         return [{ path: "specification/test-lint/test/test-lint-result.json" }];
```
Comment (raych1) [inline src/tests/momentOfTruthTest.ts:57]: Why does this constant value return need to be async?

File: src/momentOfTruth.ts (hunk 1 above)
Comment (jianyexi) [reply src/momentOfTruth.ts:183]: `getFilesChangedInPR` will filter example json, while `getChangedFilesFromPR ` not

File: src/tests/momentOfTruthTest.ts (hunk 2 above)
Comment (jianyexi) [reply src/tests/momentOfTruthTest.ts:57]: removed it

File: src/momentOfTruth.ts (hunk 1 above)
Comment (raych1) [reply src/momentOfTruth.ts:183]: Can you rename them to reflect the differences?

Comment (jianyexi) [reply src/momentOfTruth.ts:183]: I 'll remove the `getChangedFilesFromPR ` function as it hasn't been referred any more

**Pull request merged**